doc/source/working_with_images/grub_authorization.rst

```diff
@@ -0,0 +1,70 @@
+.. _grub_authorization:
+
+Authorized access to boot menu entries
+======================================
+
+.. sidebar:: Abstract
+
+    This page provides information how to configure the GRUB boot laoder
+    to allow authorized access to its boot menu entries
+
+
+You can configure &grub; to allow access to boot menu entries depending
+on the level of authorization. You can configure multiple user accounts
+protected with passwords and assign them access to different menu entries.
+To configure authorization in GRUB;, follow these steps:
+
+1. Create and encrypt one password for each user account you want to use in
+   GRUB. Use the `grub2-mkpasswd-pbkdf2`:
+
+   .. code:: bash
+
+      $ sudo grub2-mkpasswd-pbkdf2
+      Password: ****
+      Reenter password: ****
+      PBKDF2 hash of your password is grub.pbkdf2.sha512.10000.9CA4611006F77A...
+
+2. Delete the content of the `/etc/grub.d/10_linux` file and save it.
+   This prevents outputting the default GRUB menu entries.
+
+3. Edit the `/boot/grub2/custom.cfg` file and add custom
+   menu entries manually. The following template is just an example, adjust
+   it to better match your use case:
+
+   .. code:: bash
+
+      set superusers=admin
+      password admin ADMIN_PASSWORD
+      password maintainer MAINTAINER_PASSWORD
+
+      menuentry 'Operational mode' {
+        insmod ext2
+        set root=hd0,1
+        echo 'Loading Linux ...'
+        linux /boot/vmlinuz root=/dev/vda1 $GRUB_CMDLINE_LINUX_DEFAULT $GRUB_CMDLINE_LINUX mode=operation
```

Comment:
KIWI runs some patches over the generated grub.cfg to ensure it is EFI/BIOS bootable and to provide a proper `root=` kernel parameter. We need to check this template does not interfere over the grub2 configuration steps neither produce inconsistent data.